src/views/Admin/Modules/Records/List.vue

```diff
@@ -88,14 +89,20 @@ export default {
 
       return undefined
     },
+
+    moduleFields () {
```

Comment:
If this is only used once we can move it into the created method (less watchers = better)